proparse/src/main/java/org/prorefactor/core/IConstants.java

```diff
@@ -60,7 +60,7 @@
   /** See Proparse documentation, "Node Attributes Reference". */
   public static final int FULLTEXT = 1800;
   /** See Proparse documentation, "Node Attributes Reference". */
-  public static final int FROM_USER_DICT = 1900;
+  // public static final int FROM_USER_DICT = 1900;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)